multigrid/input2d.var_theta.var_vel

```diff
@@ -1,40 +1,60 @@
-USE_PRECOND = TRUE
+USE_PRECOND = FALSE
 w = 0.75
-C = 0.0
-D = 1.0
-IS_VEL_NULLSPACE = TRUE
+C = 1.0
+D = -1.0
```

Comment:
For target multigrid, I think D should be 1.0.

Reply:
It shouldn't matter because we are just solving a system. Setting `D=-1.0` is more in the line for how the time stepping scheme works.